### PR TITLE
Filter all

### PR DESCRIPTION
**Pull request by adsantos**
A spec for filtering all the items.

### CODE REVIEW

**Comment by adsantos** [comment]
The test data for this spec is the same as the one in the 'Date filter spec' PR.

**Comment by adsantos** [comment]
Ready for review

File: spec_all.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                            "date": null,
+                            "project" : null,
+                            "items": [1, 2, 3, 4, 5, 6, 7],
+                            "isReorderable": false,
+                            "labels": null
+                        }
+                    ]
+                }
+            ]
+        }
```
Comment (Garee) [inline spec_all.json]: Only other test I can think of is perhaps one for mixed-case i.e. `AlL`?

Comment (adsantos) [inline spec_all.json]: I think testing lower-case and upper-case is enough, given that the implementation should transform the input into a format to test the filter. The goal of the test is to ensure that the transformation is done.

**Pull request merged**